src/Miunie.Discord/DSharpClient.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+using DSharpPlus;
+
+namespace Miunie.Discord
+{
+    public class DSharpClient
+    {
+        private DiscordClient _discordClient;
+
+        public DSharpClient(DiscordClient discordClient)
+        {
+            _discordClient = discordClient;
+        }
+
+        public async Task ConnectAsync()
+        {
+            await _discordClient.ConnectAsync();
```

Comment:
This class is redundant. Let's get rid of it. The `DSharpClient` in the Architecture drawing was just to illustrate that we will use the Client from DSharp.